docs/noao/noao.rst

```diff
@@ -0,0 +1,18 @@
+.. doctest-skip-all
+
+.. _astroquery.noao:
+
+********************************
+NOAO Queries (`astroquery.noao`)
+********************************
+
+Getting started
+===============
```

Comment:
Need some documentation here!

Reply:
I would like to be able to skate through with minimal documentation for this release.  Yes, this is less than minimal.  We have more documentation online so I'll likely point to that if I can figure out how to add links to these docs.  Will you let minimal pass?  I have another version in the works that will add SIGNIFICANTLY MORE search capabilities.  That would be a good time to document heavily.

Reply:
yes, minimal is acceptable as a first pass, but at least a few basic examples - minimum one example per function - is needed.